plugins/admin/pkg/command/registry/add_test.go

```diff
@@ -110,7 +175,7 @@ func TestNewRegistryAddCommand(t *testing.T) {
 
 	})
 
-	t.Run("adding registry secret for service account already have imagepullsecrets", func(t *testing.T) {
+	t.Run("adding registry secret for service account that already have imagePullSecrets", func(t *testing.T) {
```

Comment:
"adding registry secret for service account that already <del>have</del> has imagePullSecrets"

Reply:
Thanks, done.